jobs/BAML_LON07506_AVP_Software Developer_ London

```diff
@@ -0,0 +1,54 @@
+---
+title: AVP, Software Developer, Global Technology and Operations
+company: Bank of America Merrill Lynch
+url: http://ig12.i-grasp.com/fe/tpl_bankofamerica06.asp?newms=jj&id=103344&aid=16952
+location: London
+contract: permanent
+contact:
+    name: Janos Meszaros
+    email: janos.meszaros@baml.com
+    ...: ...
+created: !!timestamp '2015-03-24'
+tags:
+  - london
+  - england
+  - uk
+  - python
+  - c++
+  - sql
+---
+
+# This is a header
```

Comment:
I think I have done it. We will see ☺

From: Steve Stagg [mailto:notifications@github.com]
Sent: 26 March 2015 15:45
To: pythonjobs/jobs
Cc: Meszaros, Janos
Subject: Re: [jobs] Create new file (#69)


In jobs/BAML_LON07506_AVP_Software Developer_ London<https://github.com/pythonjobs/jobs/pull/69#discussion_r27225043>:

> +contract: permanent

> +contact:

> +    name: Janos Meszaros

> +    email: janos.meszaros@baml.com<mailto:janos.meszaros@baml.com>

> +    ...: ...

> +created: !!timestamp '2015-03-24'

> +tags:

> +  - london

> +  - england

> +  - uk

> +  - python

> +  - c++

> +  - sql

> +---

> +

> +# This is a header

You should be able to click the pencil icon here:
https://github.com/pythonjobs/jobs/pull/69/files#<https://github.com/pythonjobs/jobs/pull/69/files>
(Right hand side of the page, near the top)

And edit this file online, to change it. Unfortunately I can't do this as it's 'your' request.

—
Reply to this email directly or view it on GitHub<https://github.com/pythonjobs/jobs/pull/69/files#r27225043>.

----------------------------------------------------------------------
This message, and any attachments, is for the intended recipient(s) only, may contain information that is privileged, confidential and/or proprietary and subject to important terms and conditions available at http://www.bankofamerica.com/emaildisclaimer.   If you are not the intended recipient, please delete this message.